### PR TITLE
bug solved email is now not required

### PR DESCRIPTION
**Pull request by jimitpatel9**

### CODE REVIEW

File: client/vivace/details/controller.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 module.exports = function($location, CandidateModel, $routeParams) {
 
-  let promise = CandidateModel.get({id: $routeParams.id}).$promise;
-
-  promise.then((data) => {
+  this.user = CandidateModel.get({id: $routeParams.id});
+  console.log(this.user);
```
Comment (jackofseattle) [inline client/vivace/details/controller.js]: console  and commented code - neither should be checked in.

File: client/vivace/add-edit/template.html [hunk 2]
```diff
@@ -22,7 +22,7 @@
     <md-input-container flex md-is-error="candidateForm.email.$invalid && (candidateForm.$submitted || candidateForm.email.$dirty)">
 
       <label>Email</label>
-      <input name="email" ng-required="true" ng-model="candidate.details.email" ng-pattern="candidate.regex.EMAILREGEX">
+      <input name="email" ng-required="false" ng-model="candidate.details.email" ng-pattern="candidate.regex.EMAILREGEX">
```
Comment (jackofseattle) [inline client/vivace/add-edit/template.html]: No need to set as false - just remove the ng-required directive here.

File: client/vivace/details/controller.js [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
+  /*promise.then((data) => {
     this.user = data;
     this.user.name = data.firstName + ' ' + data.lastName;
   }, () => {
     $location.path('/not-found');
-  });
+  });*/
```
Comment (prashantpawar) [inline client/vivace/details/controller.js]: To add to what Jack wrote. We use git repo to track code history, we don't need commented code checked in.

**Comment by prashantpawar** [comment]
Otherwise it looks good to me.

**Comment by jimitpatel9** [comment]
all changes completed

**Pull request merged**